### PR TITLE
update packages

### LINKED ISSUE #38: @web3-ui/core package error
I tried npx create-dao today and found an error with using the @web3-ui/core package 0.4.0.  

When using npx create-dao, v 0.4.0 is installed automatically
When using this repo in development, 0.2.0 is in the [package.json](https://github.com/Developer-DAO/create-dao/blob/966bc0c651ec2a8f693b2bcbec8d2b2317afd838/packages/create-dao/templates/default/package.json#L25)

Is this an issue with @web3-ui/core package 0.4.0?

web3-ui-error

work around is to lock down the version the user installs to v 0.2.0 [here](https://github.com/Developer-DAO/create-dao/blob/966bc0c651ec2a8f693b2bcbec8d2b2317afd838/packages/create-dao/create-app.js#L95)

`    'framer-motion@^4',
    'react-icons',
    '@web3-ui/core',
  ];
`
to :
`    'framer-motion@^4',
    'react-icons',
    '@web3-ui/core@0.2.0',
  ];
`

@Dhaiwat10 thoughts?

### PR DESCRIPTION
**Pull request by e-roy**
update packages to reflect dev for the end user and fixed errors on mint page

### CODE REVIEW

File: packages/create-dao/templates/default/pages/mint.js [hunk 1]
```diff
@@ -17,23 +17,19 @@ const NFTContractAddress = '0xD506BE3aC0345d9FaE8C2A58Ac2F02E582E68781';
 
 export default function Mint() {
   const toast = useToast();
-  const { connected, provider } = useWallet();
+  const { connected, provider, correctNetwork } = useWallet();
   const nftContract = useContract(NFTContractAddress, NFT_ABI.abi);
```
Comment (Dhaiwat10) [inline packages/create-dao/templates/default/pages/mint.js]:
```suggestion
  const [nftContract, isReady] = useContract(NFTContractAddress, NFT_ABI.abi);
```

File: packages/create-dao/templates/default/pages/mint.js [hunk 2]
```diff
@@ -17,23 +17,19 @@ const NFTContractAddress = '0xD506BE3aC0345d9FaE8C2A58Ac2F02E582E68781';
...
+  const [mintError, setMintError] = useState(null);
 
   const fetchData = async () => {
     const [maxSupply, totalSupply, mintPrice] = await Promise.all([
-      await nftContract.maxSupply(),
-      await nftContract.totalSupply(),
-      await nftContract.cost(),
+      await nftContract[0].maxSupply(),
+      await nftContract[0].totalSupply(),
+      await nftContract[0].cost(),
```
Comment (Dhaiwat10) [inline packages/create-dao/templates/default/pages/mint.js]: You can just revert these changes then

**Review by Dhaiwat10 — CHANGES_REQUESTED** [review]
Couple of changes and we're good to go!

**Comment by e-roy** [comment]
updated web3-ui package to 0.5.1 and tested

**Review by Dhaiwat10 — APPROVED** [review]

**Pull request merged**